packages/terra-i18n/tests/nightwatch/i18n-spec.js

```diff
@@ -12,6 +12,8 @@ const defaultTranslations = enTransaltions;
 
 const screenshot = require('terra-toolkit').screenshot;
 
+const pauseInms = 1000;
```

Comment:
Thanks, Ryan's help. Update to use `expect ... before` syntax to get rid of the `pause` function.